examples/android/app/src/main/jni/main.cpp

```diff
@@ -90,11 +90,11 @@ int main(int argc, char* argv[])
     image.setPosition(sf::Vector2f(screen.size) / 2.f);
     image.setOrigin(sf::Vector2f(texture.getSize()) / 2.f);
 
-    sf::Font font;
-    if (!font.loadFromFile("tuffy.ttf"))
+    std::optional<sf::Font> font = sf::Font::loadFromFile("tuffy.ttf");
+    if (!font.has_value())
```

Comment:
This could simply be `if (!font)`, no?

Might be more consistent if we later use expressions like `if (std::optional<T> var = ...)`.